core/examples/luigi-sample-angular/e2e/tests/tets.spec.js

```diff
@@ -0,0 +1,220 @@
+describe('Luigi Sample Application', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:4200');
+  });
+
+  describe('Login', () => {
+    it('Login', () => {
+      cy.login('tets@email.com', 'tets');
+      cy.get('.fd-global-nav').contains('Overview');
+      cy.location().should(loc => {
+        expect(loc.hash).to.eq('#/overview');
+      });
+    });
+  });
+
+  describe('Navigation', () => {
+    it('Click around using navigation', () => {
+      cy.login('tets@email.com', 'tets');
+
+      //overview page
+      cy.get('.fd-global-nav').contains('Overview');
+      cy.location().should(loc => {
+        expect(loc.hash).to.eq('#/overview');
+      });
+      cy.get('.fd-ui__header')
+        .contains('Projects')
+        .click();
+
+      //projects page
+      cy.get('.fd-app__sidebar').should('contain', 'Project One');
+      cy.get('.fd-app__sidebar').should('contain', 'Project Two');
+      cy.get('.fd-app__sidebar')
+        .contains('Project One')
+        .click();
+
+      //project one page
+      cy.location().should(loc => {
+        expect(loc.hash).to.eq('#/projects/pr1');
+      });
+      cy.get('.fd-app__sidebar').should('not.contain', 'Project One');
+      cy.get('.fd-app__sidebar').should('contain', 'Miscellaneous2');
+      cy.get('.fd-app__sidebar')
+        .contains('Default Child Node Example')
+        .click();
+
+      //default child node example
+      cy.location().should(loc => {
+        expect(loc.hash).to.eq('#/projects/pr1/dps/dps2');
+      });
+      cy.get('.fd-app__sidebar').should('contain', 'First Child');
+      cy.get('.fd-app__sidebar').should('contain', 'Second Child');
+    });
+  });
+
+  describe('Luigi client features', () => {
+    it('linkManager features', () => {
```

Comment:
I agree that we could split tests for 4 smaller ones (Logi, Logout, Navigation, Luigi client features), but I wouldn't split them too much, because at the beginning of each test, the app is opened again and we need to login, go to the specific page etc. That would cause that each feature is tested separately and that wouldn't find bugs that might have been caused when they are combined, and I believe this situation might happen.